episodes/introduction.md

```diff
@@ -30,17 +30,59 @@ Objectives
 
 ## Introduction
 
-Placeholder text. 
+Placeholder text.
+
+:::{image} ../media/episodes/introduction/hub-overview.png
+:width: 100%
+:alt: Diagram showing an overview of a 2i2c Hub, includiing authentication, online content, custom environments, the community hub, cloud infrastructure, support and services.
+
+:::
 
 ## Exercises
 
-::::{dropdown} Challenge Title
+::::{dropdown} Which of the following statements are True/False?
 :open:
 
-Challenge content
+a. The infrastructure for running a 2i2c-managed hub is closed source and not replicable across different cloud providers.  
+b. Access to the hub can be controlled with authentication and authorisation.  
+c. Software environments are inflexible and irreproducible.  
+d. 2i2c provides operations support for your hub.
 
 :::{dropdown} Solution
-Solution content
+a. False – Hubs are built entirely with open source and community-driven tooling. The Right to Replicate gives communities the right to replicate their infrastructure in its entirety elsewhere.  
+b. True – Hub Champions can control who can access the hub through GitHub and other authentication methods.  
+c. False – Software environments are managed by Hub Champions and automatically deployed with containerisation, allowing for scalability across the cloud and reproducibile user environments.  
+d. True – 2i2c engineers maintain service availability, uptime and operational upgrades. Hub Champions are responsible for hub configuration and management to support their user community.
+
+:::
+
+::::
+
+:::{dropdown} What are the needs of your user community?
+:open:
+
+In pairs, discuss and share the following points:
```

Comment:
Share how? Will they take notes in a shared doc? Summarise their partner's answers in a report-out afterwards? I recommend to be specific about the format/structure of the exercise. If nothing else, it will affect the time required for the activity.

Reply:
Resolved in a97784112952652c2a394deec9be71139592e2dd